tests/core/test_tracklist.py

```diff
@@ -295,3 +295,41 @@ def test_load_invalid_type(self):
 
     def test_load_none(self):
         self.core.tracklist._load_state(None, None)
+
+
+class TracklistAddTest(unittest.TestCase):
+    def setUp(self):
+        config = {"core": {"max_tracklist_length": 10000}}
+
+        tracks = [
+            Track(uri="dummy1:a", name="foo"),
+            Track(uri="dummy1:b", name="foo"),
+            Track(uri="dummy1:c", name="bar"),
+        ]
+
+        def lookup(uris):
+            return {u: [t for t in tracks if t.uri == u] for u in uris}
+
+        self.core = core.Core(config, mixer=None, backends=[])
+        self.core.library = mock.Mock(spec=core.LibraryController)
+        self.core.library.lookup.side_effect = lookup
+
+        self.core.playback = mock.Mock(spec=core.PlaybackController)
+
+        self.tl_tracks = self.core.tracklist.add(uris=[t.uri for t in tracks])
+
+    def test_add_appends_by_default(self):
+        tl_track = self.core.tracklist.add(uris=["dummy1:a"])[0]
+        all_tracks = self.core.tracklist.get_tl_tracks()
+        assert all_tracks[3].tlid == tl_track.tlid
+
+    def test_add_inserts_after_supplied_position(self):
+        tl_track = self.core.tracklist.add(at_position=1, uris=["dummy1:b"])[0]
+        all_tracks = self.core.tracklist.get_tl_tracks()
+        assert all_tracks[1].tlid == tl_track.tlid
+
+    def test_add_inserts_after_currently_playing_track(self):
+        self.core.playback.get_current_tl_track.return_value = self.tl_tracks[0]
+        tl_track = self.core.tracklist.add(at_position=-1, uris=["dummy1:c"])[0]
```

Comment:
Can you use your constant here instead?

Reply:
Oops, thanks. I shouldn't tackle these tasks when I'm tired.